### ISSUE TITLE
Build present value test notebook

### ISSUE THREAD

**Issue by dpaiton** [delvtech/agent0#864]
Building off of the [hyperdrive issue](https://github.com/delvtech/hyperdrive/issues/375), the notebook should leverage the test-fixture & agent0 frameworks. We want to produce a sequence of known trades, then conduct an exhaustive test of how individual actions impact the present value estimate for LPs.

PR here: https://github.com/delvtech/hyperdrive-experiments/pull/2

**Comment by dpaiton** [comment]
calculatePresentValue does not exist in the rust-sdk, so I created #956 for that task. Once that is complete I can continue with this one. This pushes back the expected deadline for the present value analysis to after the trading competition (17th)

**Comment by slundqui** [comment]
Currently working on the workflow on an interactive agent0 to run experiments like this:
https://github.com/delvtech/agent0/pull/1101

**Comment by dpaiton** [comment]
On hold until the present value overhaul is committed

**Comment by dpaiton** [comment]
Per my comment [here](https://github.com/delvtech/hyperdrive/issues/375#issuecomment-1885333926), we are going to can this task in favor of adding PV to the path independence check. The original question is no longer relevant bc PV should be mostly invariant.
